support-workers/src/typescript/services/salesforce.ts

```diff
@@ -5,38 +5,59 @@ import type { GiftRecipient, Title, User } from '../model/stateSchemas';
 import type { SalesforceConfig } from './salesforceClient';
 import { SalesforceClient } from './salesforceClient';
 
-export type ContactRecordRequest = {
-	IdentityID__c: string;
-	Email: string;
-	Salutation?: Title | null;
+//RecordType field in salesforce to distinguish buyer contacts from recipient contacts.
+//e.g. for contacts with more than print subscription, sent to different addresses. Has evolved to include gift recipient contacts.
+export const salesforceDeliveryOrRecipientRecordTypeId = '01220000000VB50AAG';
+
+export type BaseContactRecordRequest = {
 	FirstName: string;
 	LastName: string;
-	OtherStreet?: string | null;
-	OtherCity?: string | null;
+};
+//OtherState and MailingState properties are optional because they are mandatory for US/CAN/AUS but not collected for UK/NZ
```

Comment:
Is this comment correct? I don't see a `MailingState` property in this type?

```suggestion
//OtherState and OtherPostalCode properties are optional because they are mandatory for US/CAN/AUS but not collected for UK/NZ
```